style.css

```diff
@@ -2514,17 +2524,11 @@ ul {
   }
 }
 
-.post-sidebar h5 {
+.post-sidebar-title {
+  font-size: 18px;
   font-weight: 600;
 }
 
-@media (min-width: 1024px) {
```

Comment:
Why don't we need these border anymore?

Reply:
It wasn't being used as there was no `h5` inside `.post-sidebar`

Reply:
Ah, I see. 👍